packages/app-shell-ui/src/providers/BannerProvider.tsx

```diff
@@ -149,12 +134,4 @@ export const BannerProvider = ({ children }: BannerProviderProps) => {
   );
 };
 
-export const useBannerContext = () => {
```

Comment:
why remove this from here? there's a tight coupling between these, and these are unrelated changes - please revert